vendors/pc/boards/windows/aws_demos/config_files/FreeRTOSConfig.h

```diff
@@ -135,7 +135,7 @@ extern void vLoggingPrintf( const char * pcFormat,
 #define configPRINTF( X )    vLoggingPrintf X
 
 /* Non-format version thread-safe print. */
-#define configPRINT_STRING( X )    vLoggingPrint( X )
+#define configPRINT_STRING( X )    vLoggingPrintf( X )
```

Comment:
Why is this being changed back? What are the ramifications of this change?

Reply:
Didn't realize that there was a non-format function available. This config lacked the declaration of the function causing an MSVC warning (of `vLoggintPrint` being undefined). 
Have added the declaration and reverted to `vLoggingPrint`